working-with-your-data/working-with-your-own-data.md

```diff
@@ -114,6 +118,63 @@ wget --user=<USERNAME> --ask-password <URL>
 ```
 Using the `--ask-password` will prompt you to enter your password.
 
+### Upload large files (> 1Gb) from your own computer
+
+#### Install FileZilla
+
+Go to [FileZilla's website](https://filezilla-project.org/download.php?type=client) to download the FileZilla Client for your respective operating system.
+
+Click the big green `Download` button.
+
+Click `Download` on this next page for `FileZilla` this is the only free option but will have the functionality you need.
+
+<img src="screenshots/filezilla-download.png">
+
+After download is complete, you'll find the `FileZilla_3.48.1_<OPERATING_SYSTEM_TAG>.app.tar.bz2` in your download files or you can click on it in the corner of your web browser's screen.
```

Comment:
```suggestion
After download is complete, you'll find the `FileZilla_` file in your download files or you can click on it in the corner of your web browser's screen.
On a Mac, this will be a a `.app.tar.bz2`, on a Windows, it will be a `.exe` file. 
```